src/components/ActionButton.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import React from "react";
+import { StyleSheet } from "react-native";
+import type { StyleObj } from "react-native/Libraries/StyleSheet/StyleSheetTypes";
+import Text from "./Text";
+import Touchable from "./Touchable";
+import { whiteColor } from "../constants/colors";
+
+type Props = {
+  label: string,
+  onPress?: Function,
+  style?: StyleObj
+};
+
+const ActionButton = ({ label, onPress, style }: Props) => (
+  <Touchable
+    onPress={onPress}
+    style={style}
+    accessibilityTraits={["button"]}
+    accessibilityComponentType="button"
```

Comment:
Touchable defaults to button. So both accessibility attributes are not required.